core/assets/bundles/bundle.properties

```
@@ -1314,4 +1315,26 @@ block.spectre.description = A massive dual-barreled cannon. Shoots large armor-p
 block.meltdown.description = A massive laser cannon. Charges and fires a persistent laser beam at nearby enemies. Requires coolant to operate.
 block.repair-point.description = Continuously heals the closest damaged unit in its vicinity.
 block.segment.description = Damages and destroys incoming projectiles. Laser projectiles are not targeted.
-
+block.resupply-point.description = Refills the ammo supply of units using power.
+block.parallax.description = Fires a tractor beam at air units, making it more difficult for them to fly away.
```

maybe there is also some need for a "reeling them in" or "pulling them in", and the last part feels mildly self explanitory

```suggestion
block.parallax.description = Fires a tractor beam at enemy air units, making it more difficult for them to fly away.
```